### PR TITLE
A unified experiment system for Bento

### PR DESCRIPTION
**Pull request by dvoytenko**

### CODE REVIEW

File: extensions/amp-accordion/1.0/amp-accordion.js [hunk 1]
```diff
@@ -92,8 +92,9 @@ class AmpAccordion extends PreactBaseElement {
   /** @override */
   isLayoutSupported(unusedLayout) {
     userAssert(
-      isExperimentOn(this.win, 'amp-accordion-bento'),
-      'expected amp-accordion-bento experiment to be enabled'
+      isExperimentOn(this.win, 'bento') ||
+        isExperimentOn(this.win, 'bento-accordion'),
+      'expected "bento" or "bento-accordion" experiment to be enabled'
```
Comment (caroqliu) [inline extensions/amp-accordion/1.0/amp-accordion.js:97]: Not sure if people will know which to use from this warning message. What do you think of this suggestion?
```suggestion
      'expected global "bento" or specific "bento-accordion" experiment to be enabled'
```

Alternatively
 - only include the specific one
 - also include an amp.dev link that would explain the difference between using one or the other

File: src/service/resources-impl.js [hunk 2]
```diff
@@ -225,8 +225,9 @@ export class ResourcesImpl {
     divertIntersectResources(this.win);
 
     if (
-      getExperimentBranch(this.win, INTERSECT_RESOURCES_EXP.id) ===
-        INTERSECT_RESOURCES_EXP.experiment &&
+      (isExperimentOn(this.win, 'bento') ||
```
Comment (caroqliu) [inline src/service/resources-impl.js:228]: Would we technically want the intersect-resources exp to be on if any `bento-*` exp is on? particularly carousel and lightbox right?

**Review by caroqliu — APPROVED** [review]

File: src/service/resources-impl.js (hunk 2 above)
Comment (dvoytenko) [reply src/service/resources-impl.js:228]: I think I'd like it only for the global experiment. It's a more direct indication that bento mode is desired. But I think we'll be adjusting as we learn more.

File: extensions/amp-accordion/1.0/amp-accordion.js (hunk 1 above)
Comment (dvoytenko) [reply extensions/amp-accordion/1.0/amp-accordion.js:97]: Good idea. I modified the message as you suggest. And we can also provide a link when we have one.

**Review by krdwan — APPROVED** [review]

**Pull request merged**